addons/web/static/src/core/py_js/py_interpreter.js

```diff
@@ -388,6 +388,25 @@ export function evaluate(ast, context = {}) {
                 }
                 return result;
             }
+            case 16 /* List/Dict comprehension */: {
+                const isDict = ast.symbol === "{";
+                const result = isDict ? {} : [];
+                const iterator = _evaluate(ast.iterator);
+                for (const index in iterator) {
+                    const copyContext = {...evalContext}
```

Comment:
this seems quite bad to me, because it will evaluate all getters from the context. But we usually don't want to do that, it may be very expensive